src/frontend/AppNavigator.tsx

```diff
@@ -33,8 +32,6 @@ export const AppNavigator = ({permissionAsked}: {permissionAsked: boolean}) => {
     };
   }, []);
 
-  useReactNavigationDevTools(rootNavigationRef);
```

Comment:
just checking - removing this dep and the integration here is intentional? not against it, just making sure there's some context to this change

Reply:
It was used for debugging in a previous pr and was not taken out aftwards